vendor/github.com/newrelic/infra-integrations-sdk/jmx/jmx.go

```diff
@@ -17,14 +18,19 @@ import (
 	"time"
 )
 
+var lock sync.Mutex
 var cmd *exec.Cmd
 var cancel context.CancelFunc
 var cmdOut io.ReadCloser
 var cmdIn io.WriteCloser
 var cmdErr = make(chan error, 1)
 var done sync.WaitGroup
 
-var jmxCommand = "/usr/bin/nrjmx"
+var (
+	jmxCommand = "/usr/bin/nrjmx"
```

Comment:
Just to be aware: this will have to change to support windows-based nrjmx.

Reply:
Sure, np